src/routes/docs/products/storage/quick-start/+page.markdoc

```diff
@@ -52,9 +57,9 @@ To upload a file, add this to you web, Flutter, Apple, or Android app.
     final storage = Storage(client);
 
     final file = await storage.createFile(
-      bucketId: '[BUCKET_ID]',
+      bucketId: '<BUCKET_ID>',
       fileId: ID.unique(),
-      file: InputFile.fromPath(path: './path-to-files/image.jpg', filename: 'image.jpg'),
+      file: InputFile(path: './path-to-files/image.jpg', filename: 'image.jpg')
```

Comment:
This is deprecated as we prefer `InputFile.fromPath()`. See https://github.com/appwrite/sdk-for-flutter/blob/19ae72cf7e1a8b07915333cbfbd363e4a8098d5b/lib/src/input_file.dart#L10.

Also, please leave the trailing comma as the dart formatter prefers a trailing comma for every line.